src/vg/civcraft/mc/civchat2/command/commands/GroupChat.java

```diff
@@ -61,7 +61,7 @@ public boolean execute(CommandSender sender, String[] args){
 				return true;
 			}
 		}
-		Group group = gm.getGroup(args[0]);
+		Group group = GroupManager.getGroup(args[0]);
```

Comment:
@suirad  is there a benefit to not using gm.getGroup? if so probably should just delete the initialization https://github.com/Civcraft/CivChat2/pull/12/files#diff-a416eb617fde805bb3d1e79c0d75ffffR45